src/main/java/com/github/princesslana/eriscasper/rest/GuildRoute.java

```diff
@@ -11,22 +40,323 @@ private GuildRoute(Snowflake id) {
+   *     https://discordapp.com/developers/docs/resources/guild#get-guild-integrations</a>
+   */
+  public Route<Void, ImmutableList<Integration>> getGuildIntegrations() {
+    return Route.get(path("/integrations"), Route.jsonArrayResponse(Integration.class));
+  }
+
+  /**
+   * @see <a href="https://discordapp.com/developers/docs/resources/guild#create-guild-integration">
+   *     https://discordapp.com/developers/docs/resources/guild#create-guild-integration</a>
+   */
+  public Route<CreateGuildIntegrationRequest, Void> createGuildIntegration() {
+    return Route.post(path("/integrations"), CreateGuildIntegrationRequest.class, Void.class);
+  }
+
+  /**
+   * @see <a href="https://discordapp.com/developers/docs/resources/guild#modify-guild-integration">
+   *     https://discordapp.com/developers/docs/resources/guild#modify-guild-integration</a>
+   */
+  public Route<ModifyGuildIntegrationRequest, Void> modifyGuildIntegration(
+      Snowflake integrationId) {
+    return Route.patch(
+        path("/integrations/%s", integrationId.unwrap()),
+        ModifyGuildIntegrationRequest.class,
+        Void.class);
+  }
+
+  /**
+   * @see <a href="https://discordapp.com/developers/docs/resources/guild#delete-guild-integration">
+   *     https://discordapp.com/developers/docs/resources/guild#delete-guild-integration</a>
+   */
+  public Route<Void, Void> deleteGuildIntegration(Snowflake integrationId) {
+    return Route.delete(path("/integrations/%s", integrationId.unwrap()), Void.class);
+  }
+
+  /**
+   * @see <a href="https://discordapp.com/developers/docs/resources/guild#sync-guild-integration">
+   *     https://discordapp.com/developers/docs/resources/guild#sync-guild-integration</a>
+   */
+  public Route<Void, Void> syncGuildIntegration(Snowflake integrationId) {
+    return Route.post(
+        path("/integrations/%s/sync", integrationId.unwrap()), Void.class, Void.class);
+  }
+
+  /**
+   * @see <a href="https://discordapp.com/developers/docs/resources/guild#get-guild-embed">
+   *     https://discordapp.com/developers/docs/resources/guild#get-guild-embed</a>
+   */
+  public Route<Void, GuildEmbed> getGuildEmbed() {
+    return Route.get(path("/embed"), GuildEmbed.class);
+  }
+
+  /**
+   * @see <a href="https://discordapp.com/developers/docs/resources/guild#modify-guild-embed">
+   *     https://discordapp.com/developers/docs/resources/guild#modify-guild-embed</a>
+   */
+  public Route<GuildEmbed, GuildEmbed> modifyGuildEmbed() {
+    return Route.patch(path("/embed"), GuildEmbed.class, GuildEmbed.class);
+  }
+
+  /**
+   * @see <a href="https://discordapp.com/developers/docs/resources/guild#get-guild-vanity-url">
+   *     https://discordapp.com/developers/docs/resources/guild#get-guild-vanity-url</a>
+   */
+  public Route<Void, Optional<PartialInvite>> getGuildVanityUrl() {
+    return Route.get(
+        path("/vanity-url"),
+        response ->
+            response.code() == 204
```

Comment:
I don't see this 204 code in the docs. This has been tested?

Reply:
I forget why I did that. I'm sure I saw something, but we can change it to a clean change into PartialInvite.